### PR TITLE
Kg/csv

### PR DESCRIPTION
**Pull request by kagarmoe**

### CODE REVIEW

File: docs/resources/csv.md.erb [hunk 1]
```diff
@@ -36,12 +36,18 @@ The following examples show how to use this InSpec audit resource.
 
 <br>
 
+## Property Examples
+
+### name
+
+The `name` property tests the value of `name` as read from a CSV file versus the value declared in the test:
```
Comment (adamleff) [inline docs/resources/csv.md.erb]: I'm confused by this sentence, specifically "versus the value declared in the test" statement. Can you elaborate?

**Review by adamleff — DISMISSED** [review]

File: docs/resources/csv.md.erb (hunk 1 above)
Comment (kagarmoe) [reply docs/resources/csv.md.erb]: Clarified.

**Review by jquick — APPROVED** [review]

File: docs/resources/csv.md.erb [hunk 2]
```diff
@@ -13,7 +13,7 @@ Use the `csv` InSpec audit resource to test configuration data in a CSV file.
 A `csv` resource block declares the configuration data to be tested:
 
     describe csv('file') do
-      its('name') { should eq 'foo' }
+      its('name') { should cmp 'foo' }
```
Comment (chris-rock) [inline docs/resources/csv.md.erb:16]: 👍

**Review by chris-rock — APPROVED** [review]
@kagarmoe Thank you!

**Pull request merged**